src/site/includes/header.html

```diff
@@ -18,10 +18,9 @@
   {% include "src/site/includes/metatags.liquid" %}
 {% endif %}
 
-{% if buildtype == 'vagovstaging' %}
-<!-- Verification of site-ownership to the Insight App Sec Scanner -->
-<meta name="insight-app-sec-validation" content="ec59f99e-d869-4e3f-82d2-a61d32c65776">
-{% endif %}
+<script src="/js/record-event.js"></script>
+<script src="/js/settings.js"></script>
+{% include 'src/site/includes/google-analytics' %}
```

Comment:
`record-event` has to be before `google-analytics` in order for `recordEvent` to be defined in the subsequent capture to the `buildtype`. So, I moved all of these JS files up a bit for faster tracking.